src/MSALWrapper/AuthFlow/AuthFlows.cs

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+namespace Microsoft.Authentication.MSALWrapper.AuthFlow
+{
+    using System;
+    using System.Collections;
+    using System.Collections.Generic;
+
+    /// <summary>
+    /// A list of <see cref="IAuthFlow"/> to be executed by an <see cref="AuthFlowExecutor"/>.
+    /// </summary>
+    public class AuthFlows : IEnumerable<IAuthFlow>
+    {
+        /// <summary>
+        /// The Id of this group of auth flows.
+        /// </summary>
+        public readonly string Id;

```

Comment:
```suggestion
        public readonly string id;
```

readonly is generally lowercase

Reply:
That will trigger a StyleCop/Sonar warning - `readonly` does not impact casing preferences (in StyleCop, or generally). The rule here is that `public` properties are `camelCase`, and `protected`, `internal`, and `private` properties are `PascalCase`.

Reply:
Since this is public then should be camelCase?

Reply:
Oh I'm sorry I'm wrote that backwards, it's the reverse, all our public properties are PascalCase, internal+ are camelCase.

Reply:
I'm perfectly happy with `id` or `Id`, but I think `iD` is a bit strange. And at any rate, I think the camel casing applies to words and ID is one "word", so `id` seems reasonable by that measure. Does StyleCop really give a warning if we do `id`?

Reply:
Yes it does. I tried.